frontend/libs/studio-content-library/src/ContentLibrary/LibraryBody/pages/CodeList/hooks/useValidateUploadCodeListName.ts

```diff
@@ -0,0 +1,22 @@
+import { FileNameValidationResult } from '@studio/pure-functions';
+import { useTranslation } from 'react-i18next';
+
+export function useValidateUploadCodeListName() {
+  const { t } = useTranslation();
+
+  type FileNameValidationErrorResult = Exclude<
+    FileNameValidationResult,
+    FileNameValidationResult.Valid | FileNameValidationResult.NoRegExMatch
+  >;
+
+  const errorMessages: Record<FileNameValidationErrorResult, string> = {
+    [FileNameValidationResult.FileNameIsEmpty]: t('validation_errors.upload_file_name_required'),
+    [FileNameValidationResult.FileExists]: t('validation_errors.upload_file_name_occupied'),
+  };
+
+  const getInvalidUploadFileNameErrorMessage = (fileNameError: FileNameValidationResult) => {
+    return errorMessages[fileNameError];
+  };
+
+  return { getInvalidUploadFileNameErrorMessage };
```

Comment:
Samme her.
```suggestion
  return (fileNameError: FileNameValidationResult) => errorMessages[fileNameError];
```

Reply:
Begge steder ville jeg for øvrig også inkludert `errorMessage` i navnet på selve hooken.